ui/app/templates/components/identity/item-alias/alias-details.hbs

```diff
@@ -1,4 +1,4 @@
-<InfoTableRow @label="Name" @value={{@model.name}} @data-test-alias-name={{true}} />
+<InfoTableRow @label="Name" @value={{@model.name}} data-test-alias-name={{true}} />
```

Comment:
Curious how that @data.... got in there!

Reply:
Probably an upgrade maybe even back to the big one in 1.7

Reply:
I think it was the latest ember upgrade. I considered find-replacing all the `@data-test` instances, but we have a few cases where it looks like they're actually supposed to work that way (eg. ToolbarSecretLink 😱)